src/pages/ResetPasswordConfirm.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+import axios from "axios";
+import { Formik, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+import { Navigation } from "../components/Navigation";
+import { useParams, useNavigate } from "react-router-dom";
+import { LoadingSpinner } from "../components/LoadingSpinner";
+// import { BACKEND_ADDRESS } from "../constances";
+
+const validationSchema = Yup.object().shape({
+  newPassword: Yup.string()
+    .min(8, 'Hasło musi mieć co najmniej 8 znaków')
+    .required('Nowe hasło jest wymagane'),
+});
+
+export const PasswordResetConfirm = () => {
+  const { uid, token } = useParams();
+  const navigate = useNavigate();
+  const [isLoading, setLoading] = useState(false);
+
+  const handlePasswordResetConfirm = async (values, { setStatus }) => {
+    try {
+      setLoading(true);
+      await axios.post(`/password-reset-confirm/${uid}/${token}/`, {
+        new_password: values.newPassword,
+      });
+      alert('Hasło zostało pomyślnie zmienione.');
+      navigate('/');
+    } catch (error) {
```

Comment:
Jak mi starczy czasu, to poprawię te alerty

Reply:
Ale czy jest sens?